protocols/identify/src/protocol.rs

```diff
@@ -131,8 +131,6 @@ pub struct IdentifyInfo {
     pub listen_addrs: Vec<Multiaddr>,
     /// Protocols supported by the node, e.g. `/ipfs/ping/1.0.0`.
     pub protocols: Vec<String>,
-
-    _priv: ()
```

Comment:
So this was here for the explicit purpose of denying users to build `IdentifyInfo` structs? Or did it serve some other purpose too?

Reply:
No other purpose.